plugins/woocommerce/client/admin/client/activity-panel/activity-card/style.scss

```diff
@@ -262,9 +262,7 @@
 }
 
 .woocommerce-stock-activity-card__image-overlay__product {
```

Comment:
Should we retain a max-height rule so that it doesn't break the layout if the image aspect is very unexpected?

Reply:
I added it directly to the `ProductImage` component in d81ec4dc8d107c219a17e89227512955970ef63e. This way it affects all instances of the component.